File: java/warmup-2/arrayFront9.java
/* Given an array of ints, return true if one of the first 4 elements in the 
 * array is a 9. The array length may be less than 4.
 */
public boolean arrayFront9(int[] nums) {
    int i = 0;
    
    while(i < nums.length && i < 4) {
        if(nums[i] == 9)
            return true;
                      
        i++;
    }
                              
    return false;
}

File: python/warmup-2/array_front9.py
# Given an array of ints, return True if one of the first 4 elements in the 
# array is a 9. The array length may be less than 4.
def array_front9(nums):
    for i in xrange(4):
        if i < len(nums) and nums[i] == 9:
            return True
                  
    return False
